tests/test_e2e.py

```diff
@@ -1147,5 +1163,117 @@ def test_acquire_token_silent_with_an_empty_cache_should_return_none(self):
         #       If this test case passes without exception,
         #       it means MSAL Python is not affected by that.
 
+
+@unittest.skipUnless(broker_available, "AT POP feature is only supported by using broker")
+class PopTestCase(LabBasedTestCase):
+    def test_at_pop_should_contain_pop_scheme_content(self):
+        auth_scheme = msal.PopAuthScheme(
+            http_method=msal.PopAuthScheme.HTTP_GET,
+            url="https://www.Contoso.com/Path1/Path2?queryParam1=a&queryParam2=b",
+            nonce="placeholder",
+            )
+        result = self._test_acquire_token_interactive(
+            # Lab test users tend to get kicked out from WAM, we use local user to test
+            client_id=_AZURE_CLI,
+            authority="https://login.microsoftonline.com/organizations",
+            scope=["https://management.azure.com/.default"],
+            auth_scheme=auth_scheme,
+            )   # It also tests assertCacheWorksForUser()
+        self.assertEqual(result["token_source"], "broker", "POP is only supported by broker")
+        self.assertEqual(result["token_type"], "pop")
+        payload = json.loads(decode_part(result["access_token"].split(".")[1]))
+        logger.debug("AT POP payload = %s", json.dumps(payload, indent=2))
+        self.assertEqual(payload["m"], auth_scheme._http_method)
+        self.assertEqual(payload["u"], auth_scheme._url.netloc)
+        self.assertEqual(payload["p"], auth_scheme._url.path)
+        self.assertEqual(payload["nonce"], auth_scheme._nonce)
+
+    # TODO: Remove this, as ROPC support is removed by Broker-on-Win
```

Comment:
Is this still the case?

Reply:
It is still the case, until/unless that decision is overturned. I have pinged the forum again in its Teams channel. You are also CCed.

Currently, we keep this test case here in case that pending conversation will go another direction. But we actually skip this test case (see [line 1194](https://github.com/AzureAD/microsoft-authentication-library-for-python/pull/511/files#diff-ed3b8af48cd21dc1ac735b839d89b172a6f45569c324af70a2bdb4f60e5fbcddR1194)).

By default, I'll still proceed to merge this PR in, by EOD today.